### DISCUSSION TITLE
v2.4.0

### DISCUSSION THREAD

**Discussion by mienaiyami** [mienaiyami/yomikiru#49] [General]
**Full Changelog**: https://github.com/mienaiyami/auto-release-test/compare/v2.3.15...v2.3.16

## Changed

- Scroll to current page after switching Reader mode.
- Scroll to current page after toggling vertically fit.
- Better checking for supported extensions.
- Improved Performance.

## Added

- Ability to edit themes (no need to create new theme after changing it).
- dev:
  - Automated release

## Fixed

- Page number not changing in "click to move" mode when size is low.

<hr /><em>This discussion was created from the release <a href='https://github.com/mienaiyami/react-ts-offline-manga-reader/releases/tag/v2.4.0'>v2.4.0</a>.</em>

**Comment by omodia** [comment]
>   Better checking for supported extensions.

What does this mean?

**Reply by mienaiyami** [reply]
Its not something significant. If you look at #38 you will see that image format isn't recognized even though it runs in the app because its extension name is in all caps. I had just added capitalized extensions in checking list at the time but now I have removed the need for that by converting all ext to lower case before checking. This will make sure images opens even if they have ext names like .PNg or JPeg or something just in case.
